CONTRIBUTING.md

```diff
@@ -95,3 +95,13 @@ Entrypoint of CICD is the file .gitlab-ci.yml.
 
 There you can find the full definition of the CI/CD pipeline.
 For more details see the [CICD wiki in our Dev Guide](https://github.com/vmware/versatile-data-kit/wiki/Gitlab-CICD)
+
+# How to make a new VDK release
+
+To make a new stable release, follow these steps:
+- Create a new release from the Releases menu - this will require a new tag;
+- Autogenerate the changelog, then sort the changes alphabetically; if all change names are prefixed with the component name, this will ensure changes are grouped by component. Sorting can be done using the `sort` utility available in MacOS and Linux distros;
```

Comment:
Maybe add a link with steps how to auto-generate the changelog?

Reply:
noted, will link to the github docs to make it clearer